bin/entrypoint

```diff
@@ -2,4 +2,8 @@
 
 set -e
 
-exec pensieve run "$@"
+if [ "$1" = "rerun_config_changed" ]; then
+    exec pensieve rerun_config_changed
```

Comment:
This would be what the GCE instance would invoke to rerun all experiments with changed/new configs.